tests/specs/scss/functions.scss

```diff
@@ -0,0 +1,29 @@
+
+@include test-module('Functions') {
+
+  // Testing set functions
+  @include test('config-set [function]') {
+    @include config-set('test.bar', 'foo');
+    @include config-set('test.color', #fff);
+
+    $expect: ("test": ("bar": 'foo', "color": #fff));
+    @include assert-equal($config-attr, $expect, 'Check if map is correct.');
+
+    $config-attr: ();
+  }
+
+  // Testing get functions
+  @include test('config-get [function]') {
+    @include config-set('test.color', #000);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.